src/edu/saddleback/tictactoe/Main.java

```diff
@@ -21,8 +28,53 @@ public static void main(String[] args) {
     public void start(Stage primaryStage) {
         //Holds the entire scene, the game and the login screen
         StackPane gameStack = new StackPane();
+        StackPane loginStack = new StackPane();
+        String playerOne;
+        String playerTwo;
+
         gameStack.setAlignment(Pos.CENTER);
 
+        VBox loginBox = new VBox(20);

```

Comment:
Seems like a lot of static layout objects here - would probably be better in an FXML file?

Reply:
Will change